### PR TITLE
feat(*): disable undix

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

**Review by 0xged — APPROVED** [review]

File: src/config/constants/common.ts [hunk 1]
```diff
@@ -43,4 +43,5 @@ export const TOKEN_BLACKLIST = [
   '0x65559aa14915a70190438ef90104769e5e890a00', // OE - ENS
   '0x289ba1701c2f088cf0faf8b3705246331cb8a839', // ARBI - LPT. Disabled due to liquidity decrease
   '0x1dd5629903441b2dd0d03f76ec7673add920e765', // POLY - jEUR. Disabled due to aave not supporting anymore
+  '0x5d47baba0d66083c52009271faf3f50dcc01023c', // OP - UNIDEX. Disabled due to liquidity moved to velodrome
```
Comment (0xsambugs) [inline src/config/constants/common.ts:46]: It's UNIDX actually

File: src/home/current-positions/components/position/index.tsx [hunk 2]
```diff
@@ -596,6 +596,19 @@ const ActivePosition = ({
...
           )}
+          {(position.from.symbol === 'UNIDX' || position.to.symbol === 'UNIDX') && (
+            <StyledDetailWrapper alignItems="flex-start">
+              <Typography variant="body2" color="#db9e00" sx={{ display: 'flex', marginTop: '2px' }}>
+                <ErrorOutlineIcon fontSize="inherit" />
+              </Typography>
+              <Typography variant="caption" color="#db9e00" sx={{ display: 'flex', flex: '1' }}>
+                <FormattedMessage
+                  description="positionUNIDXNotSupported"
+                  defaultMessage="$UNIDX liquidity has beeon moved out of Uniswap, thus rendering the oracle unreliable. Swaps have been paused until a reliable oracle for $UNIDX is available"
```
Comment (0xsambugs) [inline src/home/current-positions/components/position/index.tsx:607]: You have a typo here:
`$UNIDX liquidity has been moved out of Uniswap, thus rendering the oracle unreliable. Swaps have been paused until a reliable oracle for $UNIDX is available`

File: src/position-detail/frame/index.tsx [hunk 3]
```diff
@@ -507,6 +507,16 @@ const PositionDetailFrame = () => {
             </Alert>
           </Grid>
         )}
+        {(position.from.symbol === 'UNIDX' || position.to.symbol === 'UNIDX') && (
+          <Grid item xs={12} style={{ display: 'flex', justifyContent: 'space-between', paddingBottom: '15px' }}>
+            <Alert severity="warning">
+              <FormattedMessage
+                description="positionUNIDXNotSupported"
+                defaultMessage="$UNIDX liquidity has beeon moved out of Uniswap, thus rendering the oracle unreliable. Swaps have been paused until a reliable oracle for $UNIDX is available"
```
Comment (0xsambugs) [inline src/position-detail/frame/index.tsx:515]: Same typo

**Review by 0xsambugs — COMMENTED** [review]
Found a few typos

**Pull request merged**